clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/cache/Keys.groovy

```diff
@@ -48,7 +48,8 @@ class Keys {
       case Namespace.SECRETS.ns:
         result << [
           account: parts[2],
```

Comment:
It looks like you removed account from the cache key below, so it shouldn't be available as a part here.